cordova-js-src/exec.js

```diff
@@ -60,18 +60,32 @@ module.exports = function (success, fail, service, action, args) {
             // CB-5806 [Windows8] Add keepCallback support to proxy
             onSuccess = function (result, callbackOptions) {
                 callbackOptions = callbackOptions || {};
+                var callbackStatus;
+                if (callbackOptions.status !== null) {
```

Comment:
My sincere apologies sir - you're quite right and I've amended the code to handle undefined.
I've evidently been writing too much code in too many languages at once.